### DISCUSSION TITLE
Custom Scheme URL for Search

### DISCUSSION THREAD

**Discussion by jhallam3** [soundcloud/api#302] [Q&A]
Hi, Ive got an iOS app that Ive developed and want to allow the user to click SoundCloud icon in my app to then search for a track inside SoundCloud, They currently are doing this with the url for example  'https://m.soundcloud.com/search?q=Where%20Did%20You%20Go%20Jax%20Jones%20&%20MNEK' 

This works fine if the app isn't installed iPhone browser loads the data fine, if the app is installed then Id like the user to be directed to the Search tab in the app and have the search entered and then executed, like the webpage does but in the app. 

I therefore would like to know how to use the Search option within SoundCloud via the Custom Scheme URL to allow for this too happen. 

Ive tried the specific links to tracks and that is fine it's the search options that I can't seem to find.

**Comment by dpreussler** [comment]
closing as of age, feel free to reopen if needed
